lte/gateway/c/core/oai/tasks/s1ap/s1ap_state_converter.cpp

```diff
@@ -60,6 +60,14 @@ void S1apStateConverter::state_to_proto(s1ap_state_t* state, S1apState* proto) {
     FREE_HASHTABLE_KEY_ARRAY(keys);
   }
 
+  keys                        = hashtable_ts_get_keys(&state->enbs);
+  uint32_t expected_enb_count = keys ? keys->num_keys : 0;
+  if (expected_enb_count != state->num_enbs) {
+    OAILOG_ERROR(
```

Comment:
Shouldn't this be WARNING level as well? Or do we want to explicitly call out this case as an error?

Reply:
Its an error if we are trying to write state that is inconsistent i.e. it shouldn't happen, it's a warning if the state we read is inconsistent as it is reconciling external state, is how I have been interpreting it, I am ok switching to warning if you prefer.

Reply:
Makes sense to me, let's keep them as separate, thanks